Jellyfin.Plugin.AniDB/Providers/AniDB/Metadata/AniDbSeriesProvider.cs

```diff
@@ -371,7 +371,14 @@ private async Task ParseTags(Series series, XmlReader reader)
                             if (tagSubtree.NodeType == XmlNodeType.Element && tagSubtree.Name == "name")
                             {
                                 var name = await tagSubtree.ReadElementContentAsStringAsync().ConfigureAwait(false);
-                                genres.Add(new GenreInfo { Name = name, Weight = weight });
+                                if (name == "18 restricted")
+                                {
+                                    series.OfficialRating = "NC-17";
```

Comment:
```suggestion
                                    series.OfficialRating = "XXX";
```

Looks like this category might better match the XXX content type (numerical value 1000), rather than NC-17 (numerical value 18)?

Needs testing to verify that this suggestion actually works as intended, as XXX is not mapped in the us file (default & fallback), but the 0-prefer.
https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/Localization/Ratings/us.csv
https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/Localization/Ratings/0-prefer.csv